tests/test_external_mixin.py

```diff
@@ -1,20 +1,22 @@
+import json
 import unittest
 import uuid
-import json
 from unittest.mock import MagicMock
+
+from d3a_interface.constants_limits import ConstSettings
+from d3a_interface.constants_limits import GlobalConfig
 from parameterized import parameterized
 from pendulum import now, Duration
+
+import d3a.models.strategy.external_strategies
+from d3a.constants import DATE_TIME_FORMAT
+from d3a.d3a_core.singletons import external_global_statistics
```

Comment:
I have the feeling that these imports are not optimized. Can it be?

Reply:
Thanks, you were right they were not optimized. Will do that too, thanks again!